### ISSUE TITLE
 [ONNXRuntimeError] : 9 : NOT_IMPLEMENTED : Could not find an implementation for Conv(1) node with name 'Conv_0'

### ISSUE THREAD

**Issue by Mayur28** [microsoft/onnxruntime#9685]
**Describe the bug**
I am trying to convert a pretrained pytorch VGG (.pth) model  to ONNX - I am able to create the onnx model successfully, however, when I call `model.InferenceSession`, I'm prompted with the following error:
`[ONNXRuntimeError] : 9 : NOT_IMPLEMENTED : Could not find an implementation for Conv(1) node with name 'Conv_0'`

I have tried exporting the model using various opset_versions, however, this did not help. Furthermore, I have also tried casting the model to float or double and sadly, this also does not work.

**Urgency**
None

**System information**
- OS Platform and Distribution: Windows 10 Professional, Version: 21H1, OS Build: 19043.1320 
- ONNX Runtime installed from (source or binary): Binray
- ONNX Runtime version: 1.9.0
- Python version: 3,6,8
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.2
- GPU model and memory: Nvidia GeForce GTX 1650, 4GB

**To Reproduce**
Below is the PyTorch network architecture of the VGG model that I am using.  The VGG model architecture does not contain any unusual aspects as seen below, therefore, I am surprised that onnxruntime has an issue with the Conv operation. Note that in the excerpt below, all activation functions and maxpooling operations are applied in the forward function, please let me know if I should include this as well.

Unfortunately, I am unable to attach the ONNX model that is giving the issue because github is preventing me from uploading the .pth file, or even if I put the .pth file in a .zip and upload the.zip file.

`class VGG_16(nn.Module):
    def __init__(self, pre_activation = False):
        super().__init__()
        self.pre_activation = pre_activation

        self.conv_1_1 = nn.Conv2d(3, 64, 3, stride=1, padding=1)
        self.conv_1_2 = nn.Conv2d(64, 64, 3, stride=1, padding=1)

        self.conv_2_1 = nn.Conv2d(64, 128, 3, stride=1, padding=1)
        self.conv_2_2 = nn.Conv2d(128, 128, 3, stride=1, padding=1)

        self.conv_3_1 = nn.Conv2d(128, 256, 3, stride=1, padding=1)
        self.conv_3_2 = nn.Conv2d(256, 256, 3, stride=1, padding=1)
        self.conv_3_3 = nn.Conv2d(256, 256, 3, stride=1, padding=1)

        self.conv_4_1 = nn.Conv2d(256, 512, 3, stride=1, padding=1)
        self.conv_4_2 = nn.Conv2d(512, 512, 3, stride=1, padding=1)
        self.conv_4_3 = nn.Conv2d(512, 512, 3, stride=1, padding=1)

        self.conv_5_1 = nn.Conv2d(512, 512, 3, stride=1, padding=1)
        self.conv_5_2 = nn.Conv2d(512, 512, 3, stride=1, padding=1)
        self.conv_5_3 = nn.Conv2d(512, 512, 3, stride=1, padding=1)

        # self.fc6 = nn.Linear(512 * 7 * 7, 4096)
        # self.fc7 = nn.Linear(4096, 4096)
        # self.fc8 = nn.Linear(4096, 2622)
`

**Expected behavior**
To not experience issues with the Conv operation when attempting to use onnxruntime.

**Screenshots**

**Additional context**

**Comment by faxu** [comment]
Are you using the latest version of PyTorch?

**Comment by JHB19** [comment]
Hi, i have the same problem, have you managed to overcome the obstacle somehow? I am using all packages in newest versions.

**Comment by skottmckay** [comment]
What data type are you using? There is an implementation of Conv in ORT for opset 1, however it only supports float if using the CPU Execution Provider. The error message could be more helpful if it's due to the type not being supported.

If running on GPU is an option, the CUDA execution provider that will support float, double and float16 if you register that. https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html

**Comment by JHB19** [comment]
In my case switching from Double to Float for model with some further minor fixes worked.
